### ISSUE TITLE
is32bit.exe fails on "MS-DOS" exe; non-zero exit code and the analysis fails.

### ISSUE THREAD

**Issue by emf** [cuckoosandbox/cuckoo#1357]
```
CuckooError: The package "modules.packages.exe" start function encountered an unhandled exception: Error returned by is32bit: Command '['bin\\is32bit.exe', '-f', u'C:\\Users\\USER~1\\AppData\\Local\\Temp\\x.exe']' returned non-zero exit status 1
```

the file's magic is "MZ", but it's an MS-DOS MZ, not a PE32 MZ.

Example sample:
https://virustotal.com/en/file/1b66c29349ea58b11eb49a1737c370f56ffaa72b5a42a9d985fde9bff7e09ff5/analysis/

**Comment by jbremer** [comment]
Thanks, I'm going to have a look in a bit!

**Comment by jbremer** [comment]
Closing issue as resolved. The above commit has been included since our `2.0.4` release. Thanks!
